README.md

```diff
@@ -40,6 +43,14 @@ To run the tests on your machine:
 
 Or, on the command line, run `bundle exec rake jasmine:ci`.
 
+## Releasing new versions
+
+At the moment this gem has to be released manually. There's a [story on the Reliability Engineering backlog](https://trello.com/c/gRQ8OnBl) to automate this.
+
+To release a new version, create a new Pull Request that updates [version.rb](lib/govuk_tech_docs/version.rb) and [CHANGELOG.md](CHANGELOG.md). Don't mix this in with other changes - this makes it easier to find out what what released when. See [this PR to release a new version as an example](https://github.com/alphagov/tech-docs-gem/pull/15).
```

Comment:
> find out what what released when

Should that be "find out what released when"?

Reply:
Fixed in https://github.com/alphagov/tech-docs-gem/pull/21/commits/ac94924f1a3fc7ba30ea7297c295aacaf7d39a9a